src/central-server/admin-service/int-test/src/intTest/java/org/niis/xroad/cs/test/glue/ManagementRequestsApiStepDefs.java

```diff
@@ -58,45 +62,56 @@ public class ManagementRequestsApiStepDefs extends BaseStepDefs {
     private FeignManagementRequestsApi managementRequestsApi;
 
     private Integer managementRequestId;
-
     private byte[] authenticationCertificate;
 
     @SuppressWarnings("checkstyle:MagicNumber")
-    @Step("new security server {string} authentication certificate registered")
-    public void newSecurityServerRegistered(String securityServerId) throws Exception {
+    @Step("new security server {string} authentication certificate registered from {string}")
```

Comment:
mention that this is origin, now it is unclear what "from" means if you're just reading report.